floodviz/reference_parser.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+# open up and reference file and store the relevant data in variables
+with open('floodviz/static/reference/reference.json', 'r') as f:
```

Comment:
I think this should be made into a callable function so that it's execution is explicit, rather than implicit by merely importing this python module. I think it makes sense for it to return the parsed information in a dictionary. It can then be called from floodviz/__init__.py before importing the views. You will still need to assign the result of the function to a global variable but it would be scoped to the floodviz package. The other option would be to create a python class. If you did this I would still have a function that explictly reads the configuration information from the file so the caller can control when this information is read.

I think it also makes sense to pass in the location of the reference.json as a parameter to the function. This gives us more flexibility done the road.

I think it also makes sense to put the reference.json in the examples directory for now. The build process should include a parameter for this file's location as we do with the instance/config.py.